src/core/Options.cpp

```diff
@@ -125,6 +131,15 @@ void Options::setShowStatistics(bool set)
     emit(updated(shared_from_this()));
 }
 
+void Options::enableAlphaBlending( const bool set )
```

Comment:
I think that enable is more correct, even if the other methods of this class all use get/set. People doesn't "set" alpha blending, they "enable" it. If we wanted to keep the naming fully consistent with the other methods it could be setAlphaBlendingEnabled, but it is far less readable IMO.